src/lib/gundb/UserPropertiesClass.js

```diff
@@ -133,7 +137,8 @@ export default class UserProperties {
     assign(data, properties)
 
     try {
-      await propsNode.secretAck(data)
+      propsNode.secretAck(data)
```

Comment:
1. too much code is repeating
2. have to add  .catch to propsNode.secretAck(data) because it won't be caught in catch(e) as we don't await anymore

let's add a private method:

```javascript
/**
 * @private
 */
async _storeProps(data, logLabel, logPayload = {}) {
  const { propsNode } = this
  const logError = e => log.error(`${logLabel} user props failed:`, e.message, e, logPayload)

  try {
    propsNode.secretAck(data).catch(logError)
    await AsyncStorage.setItem('props', data)
  } catch (e) {
    logError(e)
    throw e
  }
}

// usage:
await this._storeProps(data, 'set() / updateAll()', { properties })

await this._storeProps(defaultProperties, 'reset()')
```

Reply:
noted:)